blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -815,12 +822,19 @@ def treat_bone(b, parent = None):
 
   if animationFilePath:
       #save animation file
+      if modelFilePath == None:
+        if len(meshes)>1:

```

Comment:
another case where spaces could be added, please check for further cases by yourself